### DISCUSSION TITLE
Welcome to OpenSpace Discussions!

### DISCUSSION THREAD

**Discussion by alexanderbock** [OpenSpace/OpenSpace#1971] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wilddouglascounty** [comment]
Hello, Ken L. here, a lifelong observer of the night skies in Douglas County, Kansas in the United States.
Just a very specific question, one that I am asking even though I have not even downloaded OpenSpace on my laptop but have watched several of your videos that show a very interestingly dressed gentleman by the name of Carter zooming in and out of the solar system into the 3 dimensional space surrounding our solar system, all the way out to the entire galaxy.
     While I think this is fascinating, what I am looking for is different: when I walk out into my back yard and look at the 3,000 or so stars visible to my naked eye in my fairly dark Kansas sky, I see the equivalent of a dome with all the stars fixed within a 2 dimensional surface of that dome. What I'm looking to be able to do and am openly wondering if OpenSpace is capable of doing this: can it allow me to grasp the 3 dimensionality of what I see in my night sky? Can it shift that 2 dimensional scattering of stars across the dome of the sky into a volumetric depiction of the sky that I see from the ground?  Carter depicts how he can leave the solar system and get far enough away to turn that dome of constellations into a 3 dimensional space by looking back at the sun and "wiggling" or "rotating" the stars to bring out their 3 dimensional qualities, but can this software take my from-the-ground 2 dimensional constellations and "wiggle" them to show their 3 dimensionality from the perspective of where I am standing? I long to see such a volumetric depiction of the night sky I have come so familiar with my entire life, and yet I've not seen this as an option on any software, so am hoping that this is something that you can do or would consider doing in future updates. We all spend our lives on terra firma, looking up at the skies from the same perspective, so it sure would be cool to turn it into a volumetric space that reflects (obviously with a huge increase in parallax) what we know about the space that is hidden in those ancient constellations we are all so familar with.
